src/content/pt/2019/ecommerce.md

```diff
@@ -0,0 +1,639 @@
+  )
+}}
+
+{{ figure_markup(
+  image="fig20.png",
+  caption="Distribuição de bytes de terceiros (KB) por página móvel para cada plataforma de comércio eletrônico.",
+  description="Distribution of the 10, 25, 50, 75, and 90th percentiles of third-party bytes (KB) per ecommerce page for each platform. Shopify and Bigcommerce load the most third-party bytes across the distributions by more than 1,000 KB at the median.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vROHKGuMutXmxtzEoBSJNVn7DOzlfpizJh7mOkopFK8OVl_hCUHDOmKgYOojrpVsGnGWaletE7Uc5oX/pubchart?oid=1017068803&format=interactive"
+  )
+}}
+
+Plataformas como Shopify podem estender seus serviços usando JavaScript do lado do cliente, enquanto outras plataformas como Magento usam mais extensões do lado do servidor. Essa diferença na arquitetura afeta as figuras vistas aqui.
+
+Claramente, as páginas em algumas plataformas de comércio eletrônico fazem mais solicitações de conteúdo de terceiros e geram uma carga útil maior de conteúdo de terceiros. Uma análise mais aprofundada pode ser feita sobre o _motivo_ pelo qual as páginas de algumas plataformas fazem mais solicitações e têm cargas úteis de terceiros maiores do que outras.
+
+## Primeira pintura com conteúdo (FCP)
+
+{{ figure_markup(
+  image="fig21.png",
+  caption="Distribuição média de experiências FCP por plataforma de comércio eletrônico.",
+  description="Bar chart of the average distribution of FCP experiences for the top six ecommerce platforms. WooCommerce has the highest density of slow FCP experiences at 43%. Shopify has the highest density of fast FCP experiences at 47%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vROHKGuMutXmxtzEoBSJNVn7DOzlfpizJh7mOkopFK8OVl_hCUHDOmKgYOojrpVsGnGWaletE7Uc5oX/pubchart?oid=1341906463&format=interactive"
+  )
+}}
+
+[O First Contentful Paint](./performance#first-contentful-paint) mede o tempo que leva desde a navegação até que o conteúdo, como texto ou imagem, seja exibido pela primeira vez. Neste contexto, **rápido** significa FCP em menos de um segundo, **lento** significa FCP em 3 segundos ou mais e **moderado** é tudo o que está entre os dois. Observe que o conteúdo e o código de terceiros podem ter um impacto significativo no FCP.
+
+Todas as seis principais plataformas de comércio eletrônico têm pior FCP no celular do que no desktop: menos rápido e mais lento. Observe que o FCP é afetado pela capacidade do dispositivo (capacidade de processamento, memória, etc.), bem como pela conectividade.
+
+Precisamos estabelecer por que o FCP é pior no celular do que no desktop. Quais são as causas: conectividade e/ou capacidade do dispositivo ou outra coisa?
+
+## Pontuações do Progressive Web App (PWA)
+
+Consulte também o [capítulo PWA](./pwa) para obter mais informações sobre este tópico, além de apenas sites de comércio eletrônico.
```

Comment:
I think could be more fluid with "Capítulo sobre PWA" here.

Reply:
Thanks